src/descriptor/mod.rs

```diff
@@ -691,6 +768,37 @@ impl<Pk: MiniscriptKey + ToPublicKey> Descriptor<Pk> {
                     + varint_len(ms.max_satisfaction_witness_elements()?)
                     + ms.max_satisfaction_size(2)
             }
+            Descriptor::ShSortedMulti(ref smv) => {
+                let ss = smv.script_size();
+                let push_size = if ss < 76 {
```

Comment:
Since, this logic is now repeated twice. Can we make a function `push_size(usize) -> usize`?